Numeral notations for non inductive types (#12218)
coq/coq: doc/changelog/03-notations/12218-numeral-notations-non-inductive.rst

- **Deprecated**
  ``Numeral.v``, please use ``Number.v`` instead.
- **Changed**

>   the way rational and real constants are parsed.

Author: I had remembrance of @Zimmi48 telling me the contrary but couldn't find the message again, so my memory might well be fooling me.
Anyway, although both styles appear in recent changes, complete sentences are far more common, so i'l perform your suggested change, if not told otherwise.

>   the way rational and real constants are parsed.

Reviewer: I think this change request is still pending (and I'm starting another stab at reviewing, though I may not finish tonight)

  The exponent is now encoded separately from the fractional part
  using ``Z.pow_pos``. This way, parsing large exponents can no longer
  blow up and constants are printed in a form closer to the one they
  were parsed (i.e., ``102e-2`` is reprinted as such and not ``1.02``).
- **Removed**

>   OCaml parser and printer for real constants has been removed.

Reviewer 2: has -> have??
```suggestion
  OCaml parser and printer for real constants have been removed.
```

>   handled with proved Coq code.

Author: Ok, may I ask you the difference? I always thought that "proved" means "we have a mathematical proof" and "proven" means "we used it a lot, it works".

>   handled with proved Coq code.

Reviewer 3: The first source says "Proven as an adjective preceding a noun is standard in both British and American usage" (i.e., this case).

There is some variation in usage as a verb (e.g. "have proved/have proven"), "According the OED, proven is “the usual form [of the past participle] in Scottish English and also the preferred form in current North American English.”

The simple past tense is "proved".

https://www.dailywritingtips.com/proved-vs-proven/
https://www.dictionary.com/e/proved-vs-proven/

>   handled with proved Coq code.

Author: So that's more a grammatical than semantic issue, thanks!

- **Added:**

>   support for parameterized inductives and non inductives for the

Reviewer 4: ```suggestion
  support for parameterized inductive and non inductive types for the
```

  :ref:`Number Notation <number-notations>` and :ref:`String Notation
  <string-notations>` commands.
  (`#12218 <https://github.com/coq/coq/pull/12218>`_,
  fixes `#12035 <https://github.com/coq/coq/issues/12035>`_,
  by Pierre Roux, review by Jason Gross and Jim Fehrle for the

>   refman).

Reviewer 4: ```suggestion
  reference manual).
```

Author: Thanks, done
